Move Pomm support section to data providers page (#936)
api-platform/docs: core/data-providers.md

}

```

> ## Community data providers

Reviewer: ```suggestion
## Community Data Providers
```

Reviewer: Shouldn't it be at the top?

Author: :woman_shrugging:

Reviewer 2: At the bottom looks better to me!

If you don't want to use the built-in Doctrine system, alternative approaches which offer an integration with API Platform exist.

> For instance, [Pomm](http://www.pomm-project.org/) is a database access framework for PHP dedicated to PostgreSQL. The bundle to integrate it with API Platform can be found [here](https://github.com/pomm-project/pomm-api-platform).

Reviewer 2: ```suggestion
* [Pomm Data Provider](https://github.com/pomm-project/pomm-api-platform): ([Pomm](http://www.pomm-project.org/) is a database access framework dedicated to PostgreSQL database.
```

btw: https://www.w3.org/QA/Tips/noClickHere :)

Author: @dunglas Thanks for the link! ^^

ps: I'm blaming @alanpoulain on the click here :p
